politeiawww/user/user.go

```diff
@@ -21,6 +21,10 @@ var (
 	// in the database.
 	ErrSessionNotFound = errors.New("no user session found")
 
+	// ErrEmailHistoryNotFound indicates that a email history for a user
+	// was not found in the database.
+	ErrEmailHistoryNotFound = errors.New("no email history found")
```

Comment:
There's not really any reason that we would need to key off of this error.  If a email history is not found it should just return a new, empty one.

Reply:
Oh true, fixed.